aws/amazon-chime-sdk-android
# Pixel3content share artifacts

### Issue #, if available:

### Description of changes:
Pixel3 content share has artifacts due to screenshare size 

### Testing done:
verified huawei and pixel2 content share work as before
verified pixel3 content share no longer has artifacts

#### Unit test coverage
* Class coverage: 
* Line coverage: 

#### Manual test cases (add more as needed):
* [ x] Join meeting
* [ x] Leave meeting
* [ x] Rejoin meeting
* [ ] See active speaker indicator when speaking
* [ ] Mute/Unmute self
* [ ] See roster updates when remote attendees join / leave the meeting
* [x ] Enable local video
* [x ] See local video tile
* [ x] See remote video tile
* [ ] Pause remote video tile
* [ ] Resume remote video tile
* [ ] First time audio permissions
* [ ] First time video permissions

#### Screenshots, if available:
![image](https://user-images.githubusercontent.com/52674517/104041231-93fcf800-518d-11eb-923f-4fe2993dc886.png)


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
